e2e-tests/src/page-components/footer.ts

```diff
@@ -0,0 +1,15 @@
+import { Page } from '@playwright/test'
+import { PageComponentInterface } from 'src/models/page-component-interface'
+
+export default class Footer implements PageComponentInterface {
```

Comment:
This is the placeholder for page footer found on OurHealth study page. Unimplemented.